index.html

```diff
@@ -0,0 +1,318 @@
+                       </li>
+                   </ul>
+               </div>
+               <br>
+               <button type="button"><img src="images/Getmyresume.png" alt=""></button>
+               </div>
+               <div class="myskills">
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Languages.png" alt=""></li>
+                   <li class="greaterthan"><a href="">&#86;</a></li>
+                </ul>
+               </div>
+               <div class="frameworks">
+                   <ul>
+                       <li><img src="images/Frame9.png" alt=""></li>
+                       <li><img src="images/Frame10.png" alt=""></li>
+                       <li><img src="images/Frame11.png" alt=""></li>
+                   </ul>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Frameworks1.png" alt=""></li>
+                   <li class="lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Skills1.png" alt=""></li>
+                   <li class="skills-text lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+            </div>
+           </div>       
+           </section>
+
+           <section class="Contact-part">
+            <div class="headline1">
+              
+                 <h2 class="mainheader">
+                     Contact Me</h2>
+                 <p class="details-2">
+                    If you have an application you are interested in developing, 
+                    a feature that you need built or a project that needs coding. 
+                    I’d love to help with it.
+                 </p>
+               
+               <div class="form-input">
+                <form action="https://formspree.io/f/mayvkezq" method="post">
+                    <div class="container">
+                      <input type="text" placeholder="Name" name="Name" id="name" maxlength="30" required>                
+
+                      <input type="email" placeholder="Email" name="email" id="email" required>
+                  
+                      <textarea class="textbox" placeholder="Write your message here" name="comments" id="comments" maxlength="500" required></textarea>                                      
+                      
+                    </div>           
+                    <button type="submit" class="registerbtn">Get It Touch</button>    
+                    
+                  </form>
+               </div>   
+           </div>    
+           
+           </section>  
+        </main>
+</body>
+</html>
```

Comment:
- [ ] Your design should match the design from Figma.  So many differences between your design and the provided design, and We have to consider that you design still match with the design from the template.  Please use the right positioning, fonts, margin and right size to fix your both desktop and mobile design. 

Your design | Figma Design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/173096267-2a6f8cf7-0a9b-411c-8eba-66e54c8ad634.png) ![image](https://user-images.githubusercontent.com/43877689/173096734-7cff9eb4-4d32-47d4-99a1-e7b9a219ea1a.png) ![image](https://user-images.githubusercontent.com/43877689/173098027-ef22f18c-5839-4127-80d3-b6ce81daf2dc.png) | ![image](https://user-images.githubusercontent.com/43877689/173096471-0fd2f785-499d-43ed-ac83-c63377ae8539.png) ![image](https://user-images.githubusercontent.com/43877689/173096996-634fbe3b-cef3-48dd-9f97-f8c1c3be1606.png) ![image](https://user-images.githubusercontent.com/43877689/173098163-1d6d2594-54a9-4921-9f3f-3d75c28daee8.png)